.kokoro

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Display commands to stderr.
+set -x
+
+if [ ! -d .kokoro-ios-runner ]; then
+  git clone https://github.com/material-foundation/kokoro-ios-runner.git .kokoro-ios-runner
+fi
+
+pushd .kokoro-ios-runner
+git fetch > /dev/null
+TAG=$(git tag -l "v3*" | sort | tail -n1)
```

Comment:
I'm  worried about the sorting here.  "3.9" will be sorted *before* "3.11", which would be incorrect.

```bash
rsmoore:~ rsmoore$ cat versions.txt 
3.0.1
3.9
3.9.0
3.09.1
3.0
3.10.1
3.1
3.1.11
3.12.2
3.01.2
rsmoore:~ rsmoore$ sort versions.txt 
3.0
3.0.1
3.01.2
3.09.1
3.1
3.1.11
3.10.1
3.12.2
3.9
3.9.0
rsmoore:~ rsmoore$ 
```

Reply:
Yeah this is a not super ideal at the moment.

https://stackoverflow.com/questions/21394536/how-to-simulate-sort-v-on-mac-osx gives some insight on a better version sort for mac:

```
cat versions.txt | sort -t. -s -k 1,1n -k 2,2n -k 3,3n -k 4,4n
3.0
3.0.1
3.1
3.01.2
3.1.11
3.9
3.9.0
3.09.1
3.10.1
3.12.2
```

But I'll have to test this on the kokoro instance as well.